src/patternfly/components/DataList/data-list.scss

```diff
@@ -465,11 +458,26 @@
 .pf-c-data-list__expandable-content {
   max-height: var(--pf-c-data-list__expandable-content--MaxHeight);
   overflow-y: auto;
-  border-top: var(--pf-c-data-list__expandable-content--BorderTopWidth) solid var(--pf-c-data-list__expandable-content--BorderTopColor);
 
   .pf-c-data-list__expandable-content-body {
     padding: var(--pf-c-data-list__expandable-content-body--PaddingTop) var(--pf-c-data-list__expandable-content-body--PaddingRight) var(--pf-c-data-list__expandable-content-body--PaddingBottom) var(--pf-c-data-list__expandable-content-body--PaddingLeft);
 
+    > .pf-c-data-list {
+      margin-left: var(--pf-c-data-list--MarginLeft);
+    }
+
+    .pf-c-data-list__item:last-child {
+      border-bottom: 0;
+    }
+
+    .pf-c-data-list__item-row {
+      --pf-c-data-list__item-row--PaddingLeft: 0;
+    }
+
+    .pf-c-data-list__expandable-content-body {
+      --pf-c-data-list__expandable-content-body--PaddingLeft: 0;
+    }
```

Comment:
Just looking at this change from another PR - I wonder if this could be re-written as:

```
.pf-c-data-list { // this is the main selector at the root of this stylesheet
  & { // compiles to .pf-c-data-list .pf-c-data-list. I'm not sure you have to specify it's a data-list in an expandable content body, as I think that's the only use case of a nested data-list
    --pf-c-data-list__item-row--PaddingLeft: 0; // this var and the one below can be set at the component level as far as I can tell
    --pf-c-data-list__expandable-content-body--PaddingLeft: 0;

    margin-left: var(--pf-c-data-list--data-list--MarginLeft); // update the var name to indicate it's nested

    .pf-c-data-list__item:last-child {
      --pf-v5-c-data-list__item--BorderBottomWidth: 0; // if we were to do something like change this from a `border` on the last item to a border on a pseudo element of the last item or something, we would re-use this `--BorderBottomWidth` var and this override would still work.
    }    
```